.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+For questions and technical assistance, come ask the [Bokeh mailing list](https://groups.google.com/a/continuum.io/forum/#!forum/bokeh).
```

Comment:
@bryevdv do you think this is clear enough? I could also add a "The issue tracker isn't the place for general support. For questions..." or similar, but I don't want to sound mean.